src/puppetlabs/puppetdb/status.clj

```diff
@@ -0,0 +1,39 @@
+(ns puppetlabs.puppetdb.status
+  (:require [puppetlabs.puppetdb.scf.storage-utils :as sutils]
+            [puppetlabs.puppetdb.utils :as utils]
+            [puppetlabs.puppetdb.mq :as mq]
+            [puppetlabs.puppetdb.schema :as pls]
+            [schema.core :as s]))
+
+(def status-details-schema {:maintenance_mode? s/Bool
```

Comment:
I wasn't necessarily thinking we'd have a parallel pe_status, but we could. My rationale for refactoring this into a separate namespace is to limit the touchpoints between extensions and pdb.